### DISCUSSION TITLE
v1.4.0

### DISCUSSION THREAD

**Discussion by simonaronsson** [containrrr/watchtower#1197] [Releases]
## Changelog

70ca4c2 bump vulnerable packages (#1196)
be33c64 fix: move invalid token to log field (#1171)
04db93c fix: add missing portmap when needed (#1183)
a1bdd26 Fix typo on --http-api-update environment variable and add warning note for --http-api-periodic-polls (#1195)
9114447 docs: add jamesmacwhite as a contributor for doc (#1194)
8590d88 Add additional information for metrics.md (#1193)
084249c fix: linked/depends-on container restarting (#1103)
aa02d8d feat(notifications): add title field to template data (#1125)
1d59fb8 feat(notifications): support delayed sending (#1142)
2fa8a2a Fix docker-compose syntax in quick-start. GH #1105 (#1169)
93ccce8 docs: add note about docker hub private images (#1132)
ebdbe29 bump version of vulnerable dependencies (#1172)
6ff1d8a chore: bump alpine version in dockerfile (#1151)
d2f1185 docs: fix redirect link (#1146)
e14cc29 feat: improve HTTP API logging, honor no-startup-message (#1091)
b3b45ab docs: add patricegautier as a contributor for code (#1139)
b4a225c Post update time out (#1124)
782529d feat: improve session result logging (#1123)
d9d6f79 feat: use a more specific error type for no container info (#1115)
ac00e48 docs: add hypnoglow as a contributor for code (#1130)
81036b0 fix(api): return appropriate status for unauthorized requests (#1116)
c0fd77d fixing flags usage text to first capital letter. (#1102)
cb4e60c Prefer long flags in quick start example (#1029)
0060cd6 test: fully reset ghttp server (#1121)
1b405d4 test: container client tests refactor (#1117)
2c86956 test: reduce test output noise
ec43ece test: remove unused cross package dependency on mock api server
26fba69 test: refactor client tests
3bf9340 Create pull_request_template.md (#1110)
4a66a69 chore(deps): bump shoutrrr and containrd (#1088)
db13bde docs: add modem7 as a contributor for doc (#1083)
1ec5234 Update README.md (#1077)
47a640b update dependencies (sane go.mod) (#1061)
697397f feat(log): add context fields to lifecycle events (#1007)
cd0ec88 fix(notifications): default templates and logic (#1010)
fc31c6e feat(shoutrrr): update to v0.5 (#1055)
bd74c05 Use golang:1.15 in ECR credential helper example (#965)
52fe365 docs: add djquan as a contributor for doc (#1038)
c33bb05 docs: remove broken badge and fix docker-compose snippet (#1025)
9bb8991 fix(digest): check container image info for nil (#1027)
7221704 docs: link to versioned shoutrrr docs
ea300b7 ci: fix version in non-dev dockerfile
8b7b7d3 ci: fix version in dev dockerfile
45dbfca ci: add executable bit to build (#1006)
2f33620 ci: build latest-dev with script (#1005)
e396711 docs: fix note paragraph on Arguments page (#1001)
e3dd8d6 Session report collection and report templates (#981)
d0ecc23 docs: fix markdown lint issues (#995)
145fe6d Pre-update lifecycle hook (#793)
dc12a1a feat: allow hostname override for notifiers (#994)
f508c92 * feat: custom user agent (#990)
b196629 docs: add zenjabba as a contributor (#983)
4e7b1e7 Update index.md (#976)
dec6f84 test: fix metrics api test stability (#930)
61b715a docs: remove stray paragraph in notifications (#949)
91bbe74 docs: add DasSkelett as a contributor (#944)
6b155a1 Allow running periodic updates with enabled HTTP API (#916)
e308521 docs: fix more auto-format casualties (#943)
0c53b71 ci: move docs to separate action (#942)
26dbc64 Documentation updates (#936)

## Docker images

- `docker pull containrrr/watchtower:arm64v8-1.4.0`
- `docker pull containrrr/watchtower:arm64v8-latest`
- `docker pull ghcr.io/containrrr/watchtower:arm64v8-1.4.0`
- `docker pull ghcr.io/containrrr/watchtower:arm64v8-latest`
- `docker pull containrrr/watchtower:i386-1.4.0`
- `docker pull containrrr/watchtower:i386-latest`
- `docker pull ghcr.io/containrrr/watchtower:i386-1.4.0`
- `docker pull ghcr.io/containrrr/watchtower:i386-latest`
- `docker pull containrrr/watchtower:amd64-1.4.0`
- `docker pull containrrr/watchtower:amd64-latest`
- `docker pull ghcr.io/containrrr/watchtower:amd64-1.4.0`
- `docker pull ghcr.io/containrrr/watchtower:amd64-latest`
- `docker pull containrrr/watchtower:armhf-1.4.0`
- `docker pull containrrr/watchtower:armhf-latest`
- `docker pull ghcr.io/containrrr/watchtower:armhf-1.4.0`
- `docker pull ghcr.io/containrrr/watchtower:armhf-latest`

<hr /><em>This discussion was created from the release <a href='https://github.com/containrrr/watchtower/releases/tag/v1.4.0'>v1.4.0</a>.</em>

**Comment by fire1ce** [comment]
change log?

**Reply by piksel** [reply]
Chill :D It will show up as soon as the CI is done 😁

**Reply by fire1ce** [reply]
> Chill :D It will show up as soon as the CI is done 😁

ahah great! Already reading it. Thanks for you contribution!

**Comment by ksurl** [comment]
So this added the title line "Watchtower updates on `container hostname`" to each notification. Adding &title= no longer works for the notification url and I tried changing the notification template as well but it still prints it. Is there a way to customize or suppress this message like it was in 1.3.0?

**Reply by piksel** [reply]
Could you create an issue for this, preferably with what type of notification you are using. 

The `title` param is overridden by watchtower to provide the hostname in the title field. This was only automatically done for email before, but should now be supported on several services.
It should be possible to add a prefix to the title, but that seems broken (#1205). Ideally we should provide a `NOTIFICATION_TITLE_TEMPLATE`, since the hostname is available in the template data anyway...
